src/main/java/frc/robot/Constants.java

```diff
@@ -596,4 +597,39 @@ public static final class CraneConstants {
     public static final Translation2d kPositionIntake =
       new Translation2d(Units.degreesToRadians(32.042), 0.6824);
   }
+
+  public static final class ClimberConstants {
+    public static final boolean kEnable = false;
+    public static final int kMotorID = 16;
+    public static final double kHardMin = 0.0; //TODO: Set.
```

Comment:
I don't think we need a hard minimum (angle) since we're not using a limit switch to home.